src/types.ts

```diff
@@ -23,27 +23,26 @@ export interface Metadata {
 }
 
 export interface TableData {
-    columnMetadata: ColumnMetadata[];
+    columnNames: string[];
```

Comment:
What is the plan with the column metadata? we don't have any more typed columns it's all calculated by the value?

Reply:
Yup. We're not receiving the column types so all cells are autodetected